src/pkgbuild.js

```diff
@@ -768,6 +769,9 @@ export class Product {
     let incfg = {
       input: this.entry,
       plugins: [
+        replacePlugin({
+          'process.env.NODE_ENV': c.debug ? JSON.stringify('development') : JSON.stringify('production')
```

Comment:
Wonder if the calls to `JSON.stringify` can be replaced with just constants, e.g. `'”development"'`